react/features/base/conference/functions.ts

```diff
@@ -316,7 +315,7 @@ export function getOrCreateObfuscatedRoomName(state: Object, dispatch: Function)
     // stored even though a different room was joined.
     // Check if the obfuscatedRoom was already computed for the current room.
     if (!obfuscatedRoom || (obfuscatedRoomSource !== room)) {
-        obfuscatedRoom = sha512(room);
+        obfuscatedRoom = sha512(room ?? '');
```

Comment:
I know why you did this, but it reads very weird. I think we should return early if room is undefined instead.